### DISCUSSION TITLE
Can anyone help me with this?

### DISCUSSION THREAD

**Discussion by tomhostler** [NVlabs/instant-ngp#1012] [Q&A] [answered]
Hands-up, i'm  n00b!

1. Built a new PC from scratch to get into NeRF-making - not bad for life-long Mac fanboy ;-)
2. Got all the softwares (Cmake, CUDA, Python, Colmap etc) installed and running
3. Can run the Fox demo
4. When I try to use my own images with Colmap I get the following error

python /Users/NeRF/instant-ngp/scripts/colmap2nerf.py --colmap_matcher exhaustive --run_colmap --aabb_scale 16

**Traceback (most recent call last):
  File "C:\Users\NeRF\instant-ngp\scripts\colmap2nerf.py", line 19, in <module>
    import cv2
  File "C:\Users\NeRF\AppData\Local\Programs\Python\Python39\lib\site-packages\cv2\__init__.py", line 8, in <module>
    from .cv2 import *
ImportError: DLL load failed while importing cv2: %1 is not a valid Win32 application.**

After a lot of research, I have tried reinstalling Python, Colmap, Numpy etc - but can't get this to parse.

Windows 11, VS2022 and followed all the steps as closely as possible.

Any idea where I start to debug this one? 

Thanks in advance

**Comment by superhandishere** [comment]
all of the version problems

**Reply by tomhostler** [reply]
Hey, thanks for the reply - but could you expand a little more please?

I installed all the versions of Cuda, CMake, Colmap and Python as directed - alongside VS 2022 on Win 11 Pro. I also installed the optional OptiX and Vulkan SDk too.

My own research suggested Numpy versioning issues, so I tried re-installing / upgrading that library too - but same issue.

I can parse the Python for the provided 'fox' demo scene, but when I try to run the Colmap2Nerf script I get that 'invalided Win32 application error'.

Again, my research suggests its often encountered with versioning issues of Python these libraries, but as I followed the directed instructions, i'm a bit stumped as to where to start.

Any more pointers greatly appreciated

T

**Comment by superhandishere** [comment, marked as answer]
Try VS2019
Apparently this is a windows adaptive issue

tomhostler ***@***.***>于2022年10月18日 周二04:11写道：

> Hey, thanks for the reply - but could you expand a little more please?
>
> I installed all the versions of Cuda, CMake, Colmap and Python as directed
> - alongside VS 2022 on Win 11 Pro. I also installed the optional OptiX and
> Vulkan SDk too.
>
> My own research suggested Numpy versioning issues, so I tried
> re-installing / upgrading that library too - but same issue.
>
> I can parse the Python for the provided 'fox' demo scene, but when I try
> to run the Colmap2Nerf script I get that 'invalided Win32 application
> error'.
>
> Again, my research suggests its often encountered with versioning issues
> of Python these libraries, but as I followed the directed instructions, i'm
> a bit stumped as to where to start.
>
> Any more pointers greatly appreciated
>
> T
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NVlabs/instant-ngp/discussions/1012#discussioncomment-3900011>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOXVSKD2PSUWCF35YCBGQTTWDWXHJANCNFSM6AAAAAARGBAZBU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
*Pardon for what is absurd*

**Reply by tomhostler** [reply]
Hey! Thanks for the tip ... it took an entire rebuild, this time using VS2019, and a lot of grappling with various PATH dependencies along the way for various Python libraries ... but I now have it working. Thanks!

**Comment by superhandishere** [comment]
Glad it worked!

tomhostler ***@***.***>于2022年10月20日 周四22:28写道：

> Hey! Thanks for the tip ... it took an entire rebuild, this time using
> VS2019, and a lot of grappling with various PATH dependencies along the way
> for various Python libraries ... but I now have it working. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NVlabs/instant-ngp/discussions/1012#discussioncomment-3925326>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AOXVSKFKLVND3LO2E5OP2JLWEFJJ7ANCNFSM6AAAAAARGBAZBU>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
*Pardon for what is absurd*
